src/core_plugins/interpreter/server/routes/plugins.js

```diff
@@ -18,19 +18,14 @@
  */
 
 import { getPluginStream } from '../lib/get_plugin_stream';
-import { pluginPaths } from '../../../../../x-pack/plugins/canvas/server/lib/plugin_paths';
 
 export function plugins(server) {
   server.route({
     method: 'GET',
     path: '/api/canvas/plugins',
-    handler: function (request, h) {
+    handler: function (request) {
       const { type } = request.query;
 
-      if (!pluginPaths[type]) {
-        return h.response({ error: 'Invalid type' }).code(400);
```

Comment:
Why was this removed?

Reply:
we will do this check on pathsregistry level